services/src/main/java/org/keycloak/protocol/ciba/utils/EarlyAccessBlockerParser.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.protocol.ciba.utils;
+
+import java.util.Map;
+import java.util.UUID;
+
+import org.jboss.logging.Logger;
+import org.keycloak.common.util.Time;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.protocol.ciba.EarlyAccessBlockerStore;
+
+public class EarlyAccessBlockerParser {
+
+    private static final Logger logger = Logger.getLogger(EarlyAccessBlockerParser.class);
+
+    public static void persistEarlyAccessBlocker(KeycloakSession session, String id, EarlyAccessBlocker earlyAccessBlockerData, int expires_in) {
+        if (id == null) {
+            throw new IllegalStateException("ID not present in the data");
+        }
+        UUID key = UUID.fromString(id);
```

Comment:
@pedroigor I've reviewed `EarlyAccessBlockerStore` and found that we can simply use `String` instead of `UUID`. Anyway. I would like to use `UUID` or `String` without validating/converting them.